### DISCUSSION TITLE
Tuya Smart for Zigbee Solar Light and Rain Sensor 2in1 function Alarm Real-time Notification Light Sensing Rain Detector

### DISCUSSION THREAD

**Discussion by UsefulVid** [Koenkk/zigbee2mqtt#23762] [Devices]
It seems that there is a rain and sunlight sensor for zigbee finally available:
![image](https://github.com/user-attachments/assets/1c276d52-ac6a-475a-b42e-d56c93e63374)
https://www.aliexpress.com/item/1005007362071170.html?utparam-url=scene%3Asearch%7Cquery_from%3A&search_p4p_id=202408280614317680390981727360000358907_3

Couldn't find it yet in z2m ;)

**Comment by Westcott1** [comment]
Have a look here -
[https://smarthomescene.com/reviews/tuya-zigbee-solar-rain-sensor-rb-srain01-review/](url)

Mine seems to stop reporting after a while.

**Comment by nmaster2042** [comment]
I have this device and I have the Cleaning reminder property always true.
I cleaned the sensor and it stays true.
Do you know how it can be put to false, to detect the next time it needs to be cleaned ?
